projects/packages/external-media/src/features/admin/external-media-import-button.js

```diff
@@ -0,0 +1,20 @@
+import { __ } from '@wordpress/i18n';
+
+document.addEventListener( 'DOMContentLoaded', function () {
+	const addNewButton = document.querySelector( 'a.page-title-action' );
+	if ( addNewButton ) {
+		const importButton = document.createElement( 'a' );
+		importButton.className = 'button';
+		importButton.role = 'button';
+		importButton.innerHTML = __( 'Import Media', 'jetpack-external-media' );
+		importButton.href = window.JETPACK_EXTERNAL_MEDIA_IMPORT_BUTTON?.href;
+		importButton.style = `
```

Comment:
It would've been easier to just add a `page-title-action` class here. However, I can't do that because then clicking it would open the inline uploader 😭 
https://github.com/WordPress/wordpress-develop/blob/7d10dd7b0fde2a782395887c2d66439481440f9b/src/js/media/views/frame/manage.js#L43-L46

So I need to cherry-pick the styles manually...

Reply:
How about simply disabling the click event by?
```js
importButton.onclick = ( event ) => event.stopImmediatePropagation();
```

Reply:
Seems to work, thanks! I thought the original onclick handler that's set by the inline uploader will happen later after the plupload is ready 🤦